packages/cli/src/build/request.rs

```diff
@@ -3936,49 +3956,13 @@ impl BuildRequest {
 
         // Manually inject the wasm file for preloading. WASM currently doesn't support preloading in the manganis asset system
         head_resources.push_str(&format!(
-            "<link rel=\"preload\" as=\"fetch\" type=\"application/wasm\" href=\"/{{base_path}}/assets/{wasm_path}\" crossorigin>"
+            "<link rel=\"preload\" as=\"fetch\" type=\"application/wasm\" href=\"/{{base_path}}/{wasm_path}\" crossorigin>"
         ));
         Self::replace_or_insert_before("{style_include}", "</head", &head_resources, html);
 
         Ok(())
     }
 
-    /// Inject loading scripts if they are not already present
```

Comment:
The dioxus CLI will use a `index.html` file in the root of your project if it finds one. We only require `<div id="main">` in that file and manipulate it to have all the right loading logic. Could you remove the script from the dev and prod html files and move it back here so we can support that usecase?

Reply:
Should I keep the `inject_loading_scripts` flag and the `__wbindgen_start` opt out? I don't understand when you want to opt out of this and adding the script twice should just load it once if I understood the browser specification correctly.

Reply:
The `__wbindgen_start` opt out would probably need to be adjusted. Also note that the imported script now automatically loads the wasm which was not the case before. Unsure for which usage patterns this is relevant or if we simply want to make this a breaking change.